src/sanitizer.js

```diff
@@ -116,6 +115,37 @@ const BLACKLISTED_ATTR_VALUES = [
 ];
 
 
+/** @const {!Object<string, !Object<string, !RegExp>>} */
+const BLACKLISTED_TAG_SPECIFIC_ATTR_VALUES = {
+  'input': {
+    'type': /(?:image|file|password|button)/i,
```

Comment:
Input[type=button] is not allowed in validator so this is here to match that as well. I don't remember the reason why we blacklisted it but can try to look it up and see. <button> is still allowed.